editoast/src/client/group.rs

```diff
@@ -71,6 +80,36 @@ pub async fn list_group(pool: Arc<DbConnectionPoolV2>) -> anyhow::Result<()> {
     Ok(())
 }
 
+pub async fn get_group(
+    GetArgs { name }: GetArgs,
+    openfga_config: OpenfgaConfig,
+    pool: Arc<DbConnectionPoolV2>,
+) -> anyhow::Result<()> {
+    let regulator = openfga_config.into_regulator(pool).await?;
+    let driver = regulator.driver();
+    let Some(group_id) = driver.get_group_id(&name).await? else {
+        println!("No such group: '{name}'");
+        return Ok(());
+    };
+    let Some(GroupInfo { name }) = driver.get_group_info(group_id).await? else {
+        println!("No such group ID: {group_id}");
```

Comment:
```suggestion
        tracing::error!(group_id, "No such group");
```

Reply:
Done in https://github.com/OpenRailAssociation/osrd/compare/d8e159020578f8e1bdea55251513fe5cca492ccf..b5bcb14002444f3ba5aa2a45718fa551031fde65.